docs/cmdline-opts/no-clobber.d

```diff
@@ -0,0 +1,14 @@
+Long: no-clobber
+Help: Create necessary local directory hierarchy
```

Comment:
That seems like a wrong description!

Reply:
Whoops! Guess you can tell which one I used as a template